### ISSUE TITLE
Semantics for updated/new subobjects not available after page edit

### ISSUE THREAD

**Issue by mourawi** [SemanticMediaWiki/SemanticMediaWiki#5242]
### Setup

- SMW version: 4.0.1
- MW version: 1.35.5
- PHP version: 7.4.3
- DB system (MySQL, Blazegraph, etc.) and version: MySQL 8
- Page Forms: 5.4

### Issue

Observing issues with processing of subobjects (#subobject tag).

Data pages are created with Page Forms forms (with 'embed in field' option). Content of created/updated pages look good (all subobject templates are defined properly as entered in the form).

However, after saving edited page, the subobject query on these pages shows an 'old' content of subobjects (before edit). The updated content of subobjects is properly displayed only AFTER the page is REFRESHED (seems like subobjects creation/updates are deferred and not available just after edit). Obviously it is not desired from the user perspective.

How to fix that ?

PS. The same code worked fine (as we would expect, with updated subobjects properly displayed just after edit) before our migration: with MW 1.34.4, SMW 3.1.6, Page Forms: 4.9.5 ... also with Semantic Internal Object extension.

**Comment by mourawi** [comment]
Updated PHP to 7.4.29 (and reinstalled entire MW/SMW stack) ... still the same result.

**Comment by mourawi** [comment]
Tried SMW 3.2.3 ... still the same result.  

Really would appreciate any hint what is causing that.

**Comment by mourawi** [comment]
These are the steps to repeat the behavior:

**Template:Author**
> __NOCACHE__
>  {{#set:Title={{{name|}}}}}
> {{{books|}}}
>
>==Author==
>Name: {{{name|}}}
>
>==Books==
>{{#ask:[[-Has subobject::{{PAGENAME}}]]|?Title|?Description|sep=;|format=table}}

**Template:Book**
>{{#subobject:
>|Title={{{title|}}}
>|Description={{{description|}}}
>}}

**data page e.g. Ken Follet**
>{{Author
>|name=Ken Follett
>|books={{Book|title=World Without End|description= ... }},
>{{Book|title=The Pillars of the Earth|description=----}}
>}}


**Behavior**
every time I edit and save 'Ken Follet' page and add/edit one of the books
the page shows 'old' data ... teh page needs to be Refreshed to see updated book information

**Comment by mourawi** [comment]
It was a configuration issue ... closing issue.

**Comment by kghbln** [comment]
> It was a configuration issue ... closing issue.

I'd recommend to share the configuration tweaks in case others run into the same issue.
